### PR TITLE
removed hardcoded data and service invokation

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/Redux/actionCreators/ServiceActions.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
-import { Root } from "protobufjs";
```
Comment (chetankothari) [inline src/Redux/actionCreators/ServiceActions.js:1]: If we don't need this, we can remove this package as well from `package.json`

File: src/Redux/actionCreators/ServiceActions.js [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
 import { Auth, API } from "aws-amplify";
 
 import { APIEndpoints, APIPaths } from "../../config/APIEndpoints";
-import GRPCProtoV3Spec from "../../assets/models/GRPCProtoV3Spec";
```
Comment (chetankothari) [inline src/Redux/actionCreators/ServiceActions.js:5]: We can delete this file if we are not using it anymore.

File: src/Redux/reducers/ServiceDetailsReducer.js [hunk 3]
```diff
@@ -36,29 +36,14 @@ const groups = state => {
...
-  //   },
-  // };
-  // // till here
   const serviceGroups = groups(state);
   const availableGroup = find(serviceGroups, ({ endpoints }) =>
     some(endpoints, endpoint => endpoint.is_available === 1)
   );
   if (availableGroup) {
-    // return { ...availableGroup, payment };
     return { ...availableGroup };
```
Comment (chetankothari) [inline src/Redux/reducers/ServiceDetailsReducer.js]: You can just return `availableGroup`.

File: src/Redux/reducers/ServiceDetailsReducer.js [hunk 4]
```diff
@@ -36,29 +36,14 @@ const groups = state => {
...
   }
-
   const firstGroup = first(serviceGroups);
-  // return firstGroup && { ...firstGroup, payment };
   return firstGroup && { ...firstGroup };
```
Comment (chetankothari) [inline src/Redux/reducers/ServiceDetailsReducer.js]: Just return `firstGroup`

**Review by chetankothari — CHANGES_REQUESTED** [review]
These changes look good, but you also need to fetch the service details from the new endpoint. You could wire that up too else things will start breaking.

**Review by chetankothari — APPROVED** [review]

File: src/Redux/actionCreators/ServiceActions.js (hunk 1 above)
Comment (Vivek205) [reply src/Redux/actionCreators/ServiceActions.js:1]: https://github.com/singnet/refactored-dapp/pull/155/commits/bf47c0adaf2e0b838a7cf571d63c3f79f2e4b84f

File: src/Redux/actionCreators/ServiceActions.js (hunk 2 above)
Comment (Vivek205) [reply src/Redux/actionCreators/ServiceActions.js:5]: https://github.com/singnet/refactored-dapp/pull/155/commits/1f61bb215b25d6e0224f415c1aca4ee7776d682a

File: src/Redux/reducers/ServiceDetailsReducer.js (hunk 3 above)
Comment (Vivek205) [reply src/Redux/reducers/ServiceDetailsReducer.js]: https://github.com/singnet/refactored-dapp/pull/155/commits/a7a35cdc0ee7f1ca12499e1eed0ccadd62e1d527

File: src/Redux/reducers/ServiceDetailsReducer.js (hunk 4 above)
Comment (Vivek205) [reply src/Redux/reducers/ServiceDetailsReducer.js]: https://github.com/singnet/refactored-dapp/pull/155/commits/a7a35cdc0ee7f1ca12499e1eed0ccadd62e1d527

**Pull request merged**